creator-node/src/services/stateMachineManager/stateReconciliation/issueSyncRequest.jobProcessor.ts

```diff
@@ -418,12 +402,13 @@ const _additionalSyncIsRequired = async (
     )
   }
 
-  const response = { outcome }
+  const response: AdditionalSyncIsRequiredResponse = { outcome }
   if (additionalSyncIsRequired) {
     response.syncReqToEnqueue = {
-      userWallet,
-      secondaryEndpoint: secondaryUrl,
-      syncMode: SYNC_MODES.SyncSecondaryFromPrimary
+      syncType: SyncType.Recurring,
+      syncMode: SYNC_MODES.SyncSecondaryFromPrimary,
+      syncRequestParameters,
+      attemptNumber
```

Comment:
this is what should fix https://linear.app/audius/issue/CON-295/fix-sync-metric-label-has-invalid-value